### ISSUE TITLE
TypeError: freeBalance is undefined

### ISSUE THREAD

**Issue by rajdeep7Singh** [polkadot-js/api#5409]
* **I'm submitting a ...**

  - [x] Bug report

* **What is the current behavior and expected behavior?**

`api.tx.council.vote` is giving error for Equilibrium parachain `TypeError: freeBalance is undefined`

The tx should go through for council members and should give `not a member` for regular users

* **Please tell us about your environment:**

  - Version:     
    "@polkadot/api": "^9.11.1",
    "@polkadot/api-augment": "^9.11.1",
     wss://parachain-devnet.equilab.io/eq/collator/api/wss
  - Environment:
    - [x] Browser

  - Language:

    - [x] TypeScript (include tsc --version) - "typescript": "^4.4.3",

**Comment by jacogr** [comment]
Equilibrium has a very different setup to normal Substrate chains. So for instance, the team has supplied a number of enhancements to eg. the apps UI (see https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/api/spec/equilibrium.ts) to patch the internal API derives.

Out of the box the API itself can only cater for what is in base Substrate chains, i.e. known interfaces. Once chains has been modified heavily (in this case the full system -> account data structures, it just has no idea how to "automatically" deal with that and where the data lies).

TL;DR Without some injected patches in the API init, it cannot work out of the box in this case. (Which is why there are the API options to guide it along an understand the data structures. In this specific case they take the form of derives that are injected and used)

**Comment by rajdeep7Singh** [comment]
Thank you for the detailed response. Although i am providing typesBundle during api initialization would it be possible for you to tell me what else needs to be passed to make this work?
